quipucords/api/models.py

```diff
@@ -53,3 +53,16 @@ def update(self, request, *args, **kwargs):
 
     class Meta:
         verbose_name_plural = 'Host Credentials'
+
+
+class NetworkProfile(models.Model):
+    """A network profile connects a list of credentials and a list of hosts."""
+    name = models.CharField(max_length=64, unique=True)
+    # Comma-separated list of host specifiers
+    hosts = models.CharField(max_length=1024)
```

Comment:
Doesn't seem to match the swagger docs where we are looking for an array of strings. I expected we would be trying to utilize some of the Many to Many Modeling capabilities in Django (https://docs.djangoproject.com/en/1.11/ref/models/fields/#django.db.models.ManyToManyField.through_fields) so we can take an array and not a comma separated string.

Reply:
I like using many-to-many fields, but it changes the semantics from what Rho does. In Rho, a user can delete a credential that they're using for a profile and then recreate a new credential with the same name and it will be fine. Using many-to-many fields would change that. I think it makes sense to use many-to-many, but do you think the change will be a problem for people?

Reply:
Sorry, this comment was about credentials below.

Reply:
I think its fine to diverge from what `rho` does. While `rho` allowed for the above flow I'm not sure it was designed that way, but just a usable side effect. However, along the same lines it introduced an issue where users could unintentionally delete a credential and then run a scan with a profile that utilized it. I think the referential integrity is an improvement.

Reply:
Will this eventually become a one to many or many to many relationship? I mean having the size of 1024 will allow specifying at most 60 hosts considering IPv4 addresses considering IP addresses like `192.168.120.142` and separated by a comma without spaces around the comma. 

PS I have not checked the Swagger docs, I will do that!